package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "build": "babel src -d dist",
     "clean": "while read line; do rm -rf $line; done < .gitignore",
+    "deploy:docs": "gulp docs && $(npm bin)/gh-pages -d docs/build -m 'deploy docs [ci skip]'",
```

Comment:
Build and deploy docs.  This makes a flat directory with just the doc site `build` dir.  Since there are no tests (or anything else) in there, we skip ci.